lava-v2-jobs-from-api.py

```diff
@@ -71,23 +72,79 @@ def get_builds(api, token, config):
     return builds
 
 
-def add_callback_params(params, config, plan):
-    callback = config.get('callback')
-    if not callback:
-        return
+def is_callbacks_file_valid(callbacks):
+    expected_fields = ['type', 'token', 'url', 'dataset', 'method', 'content-type']
+    broken_callbacks = []
+
+    for c in callbacks:
+        if not c.get('url'):
```

Comment:
I would check whether all mandatory fields ('expected_fields') are present. There is no point checking any further if fields is missing.

Reply:
@mwasilew , only `url` field would be mandatory. The rest of them can be optional. If an expected field is not present, it will be set to LAVA's default.